requirements_all.txt

```diff
@@ -442,7 +442,7 @@ boschshcpy==0.2.28
 
 # homeassistant.components.amazon_polly
 # homeassistant.components.route53
-boto3==1.16.52
+boto3==1.20.34
```

Comment:
This is not compatible with aiobotocore==1.2.2:
```
(home-assistant) x:~/dev/home-assistant/core$ pipdeptree -w silence -r -p botocore
botocore==1.19.52
  - aiobotocore==1.2.2 [requires: botocore>=1.19.52,<1.19.53]
  - boto3==1.20.34 [requires: botocore>=1.23.34,<1.24.0]
    - pycognito==2022.1.0 [requires: boto3>=1.10.49]
      - hass-nabucasa==0.51.0 [requires: pycognito==2022.01.0]
    - pyhiveapi==0.4.2 [requires: boto3>=1.16.10]
    - pyoverkiz==1.1.0 [requires: boto3>=1.18.59,<2.0.0]
    - warrant-lite==1.0.4 [requires: boto3>=1.9,<2.0]
      - pyoverkiz==1.1.0 [requires: warrant-lite>=1.0.4,<2.0.0]
  - pyhiveapi==0.4.2 [requires: botocore>=1.19.10]
  - s3transfer==0.5.0 [requires: botocore>=1.12.36,<2.0a.0]
    - boto3==1.20.34 [requires: s3transfer>=0.5.0,<0.6.0]
      - pycognito==2022.1.0 [requires: boto3>=1.10.49]
        - hass-nabucasa==0.51.0 [requires: pycognito==2022.01.0]
      - pyhiveapi==0.4.2 [requires: boto3>=1.16.10]
      - pyoverkiz==1.1.0 [requires: boto3>=1.18.59,<2.0.0]
      - warrant-lite==1.0.4 [requires: boto3>=1.9,<2.0]
        - pyoverkiz==1.1.0 [requires: warrant-lite>=1.0.4,<2.0.0]
```

Reply:
Resolved the conflict by choosing compatible versions. Thanks for bringing it to my attention.

To help prevent these kinds of issue, I've opened #64058 which would add a `pip check` job if requirements change. That would have highlighted the issue too.
```
...
boto3 1.20.34 has requirement botocore<1.24.0,>=1.23.34, but you have botocore 1.19.52.
```

Reply:
Anything we need to do in `pyoverkiz` for this? We currently just update to the newest version asap via dependabot, however we only use a small portion for login via Cognito in pyoverkiz.

Reply:
> Anything we need to do in `pyoverkiz` for this? We currently just update to the newest version asap via dependabot, however we only use a small portion for login via Cognito in pyoverkiz.

I don't think so. From what I understand breaking changes are only introduced with a new major version. The policy is a bit vague with regards to minor versions though. https://docs.aws.amazon.com/sdkref/latest/guide/maint-policy.html

In most use cases only a small subset of the API is used anyway. Pretty likely something fairly common that isn't going to change. If you don't use `amazon_polly` or `route53`, it's also possible that you're already using a newer version of boto3 with Home Assistant.

Feel free to test it though!

_For reference, the changelog: https://github.com/boto/boto3/blob/1.20.24/CHANGELOG.rst_